packages/Meshfree/test/tstNearestNeighborExodusGenerator.cpp

```diff
@@ -188,11 +198,6 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( ExodusProblemGenerator, one_to_one, Node )
                           tgt_coord_copy );
     Kokkos::fence();
 
-    // Set an epsilon for the search.
-    auto const epsilon_default =
-        DataTransferKit::DistributedSearchTreeImpl<DeviceType>::epsilon;
-    DataTransferKit::DistributedSearchTreeImpl<DeviceType>::epsilon = 1.0;
-
```

Comment:
I actually like it better just before the operator.   Note that it is only used at construction time so you may set it back to the default value right after.